amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```
@@ -2495,7 +2497,194 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ##retention-schedule
+  '/record-categories/{categoryId}/retention-schedules':
```

The existing references to this path look like:

```suggestion
  '/record-categories/{recordCategoryId}/retention-schedules':
```

---

Changes are done as per the comment.